jujugui/static/gui/src/app/app.js

```diff
@@ -1782,9 +1782,11 @@ YUI.add('juju-gui', function(Y) {
     },
 
     _displayShortcutsModal: function() {
+      const guiVersion = GUI_VERSION; // eslint-disable-line no-undef
       ReactDOM.render(
         <window.juju.components.ModalShortcuts
           closeModal={this._clearShortcutsModal.bind(this)}
+          guiVersion={guiVersion.version}
```

Comment:
Yeah, you'll need to modify the makefile to assign it to `window.GUI_VERSION` instead of a const. Then it'll be available in both `window.GUI_VERSION` and `GUI_VERSION`